app/pathogen/arbovirus/wip/page.tsx

```diff
@@ -0,0 +1,39 @@
+"use client"
```

Comment:
The path I'm using is `http://localhost:3000/pathogen/arbovirus/wip` because this is a work in progress. I'd really love to move this to `http://localhost:3000/pathogen/arbovirus/dashboard` when we're ready to enable the feature flag but that route is obviously taken.

Anyways, if anyone has any route ideas you can just drop them here. I'm not overly concerned with how we'll do the whole route thing, we can move these files on a moments notice and nobody in production will see them right now :)